### ISSUE TITLE
url to Dataset no longer working

### ISSUE THREAD

**Issue by 13bmartens** [sarnikowski/bert_in_a_flask#8]
Accessing the dataset url gives the following error message:

<Error><Code>NoSuchBucket</Code><Message>The specified bucket does not exist.</Message></Error>

**Comment by sarnikowski** [comment]
Yes, it seems that google decided to remove the direct link to the data in `.csv` format. I will see if I can find another link.
